INSTALL

```diff
@@ -79,8 +79,8 @@ Install the following tools and repos needed for building:
 
 >>>>> Command 1b: 
 # yum install git yum-utils wget \
-    http://download.fedoraproject.org/pub/epel/6/x86_64/epel-release-6-8.noarch.rpm \
-    http://elrepo.org/linux/elrepo/el6/x86_64/RPMS/elrepo-release-6-5.el6.elrepo.noarch.rpm
+    http://mirror.math.princeton.edu/pub/epel/7/x86_64/e/epel-release-7-7.noarch.rpm \
```

Comment:
In reviewing this, perhaps the domain name ought to remain domain.fedoraproject.org